chipsec/hal/uefi_common.py

```diff
@@ -1733,7 +1733,7 @@ class EFI_CONFIGURATION_TABLE():
     def __init__( self ):
         self.VendorTables = {}
     def __str__(self):
-        return ( 'Vendor Tables:\n%s' % (''.join( ['{%s} : 0x{:016X}\n' % (vt,self.VendorTables[vt]) for vt in self.VendorTables])) )
+        return ( 'Vendor Tables:\n{}'.format(''.join( ['{}{}{} : 0x{:016X}\n'.format('{',vt,'}',self.VendorTables[vt]) for vt in self.VendorTables])) )
```

Comment:
This can be simplified to:
```python
return ( 'Vendor Tables:\n{}'.format(''.join( ['{{{}}} : 0x{:016X}\n'.format(vt,self.VendorTables[vt]) for vt in self.VendorTables])) )
```

Reply:
Done